pkg/pillar/types/errortime.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+import (
+	"reflect"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Common error with timestamp
+
+// ErrorAndTime is used by many EVE agents
+type ErrorAndTime struct {
+	Error     string
+	ErrorTime time.Time
+}
+
+// SetErrorNow uses the current time
+func (etPtr *ErrorAndTime) SetErrorNow(errStr string) {
```

Comment:
Great name.. makes it very clear

Reply:
Do you have more comments? You are still listed and changes requested, and I think I did those changes (but I expunged the old ErrorInfo name to make it easier to detect merge issues since there are PRs in flight which add/modify code.(